src/pages/Checkout.tsx

```diff
@@ -146,7 +241,8 @@ export const Checkout = () => {
           }}
         >
           <Typography variant="h3">
-            Your payment value is {formatCost(payment)}
+            Your payment value is{' '}
+            <PaymentCurrencySelector onChange={setPriceOverride} {...checkout} />
```

Comment:
So if I understood it correctly, this allows you to choose in which currency you want to pay?

Reply:
This allows you to select USD as alternative payment option (for offers which are not in USD)